src/main/java/de/tum/in/www1/artemis/web/rest/dto/UserPageableSearchDTO.java

```diff
@@ -0,0 +1,61 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class UserPageableSearchDTO<T> extends PageableSearchDTO<T> {
```

Comment:
I don't know what the exact purpose of the generic here is for, but from the looks of it you always use it with Strings in the rest of the code. Will it be always a String? Or do you need this parameter to be generic?
```suggestion
public class UserPageableSearchDTO extends PageableSearchDTO<String> {
```

Reply:
Good point - i guess we can remove the generic :D